js/load_jsons.js

```diff
@@ -84,3 +85,8 @@ function fetch_keys(device) {
 
     });
 }
+
+function get_parameter_by_name(name) {
+    var match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
+    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
```

Comment:
It's enough only 2 spaces for indentation.

Reply:
@rtakacs I will fix it.